k256/src/arithmetic.rs

```diff
@@ -24,12 +25,41 @@ use elliptic_curve::{
     weierstrass::GenerateSecretKey,
 };
 
-/// b = 7 in Montgomery form (aR mod p, where R = 2**256.
-const CURVE_EQUATION_B: FieldElement = FieldElement([
-    0x0000_0007_0000_1ab7,
-    0x0000_0000_0000_0000,
-    0x0000_0000_0000_0000,
-    0x0000_0000_0000_0000,
+const CURVE_EQUATION_B_SINGLE: u32 = 7u32;
+
+const CURVE_EQUATION_B: FieldElement = FieldElement::from_bytes_unchecked(&[
```

Comment:
Looks like `rustfmt` is... not being very helpful here. You might want to:

```suggestion
#[rustfmt::skip]
const CURVE_EQUATION_B: FieldElement = FieldElement::from_bytes_unchecked(&[
```

...and format it slightly better than what `rustfmt` is doing.

Reply:
Thanks, fixed